examples/data/ldcE.ts

```diff
@@ -1596,20 +1596,16 @@ export const validLDCE: Schema = {
     },
   ],
   metadata: {
+    source: 'PlanX',
+    organisation: 'Buckinghamshire',
+    id: '95f90e21-93f5-4761-90b3-815c673e041f',
     service: {
       flowId: '824628b2-deeb-48b0-92b1-2ca7f3b17163',
-      name: 'Apply for a lawful development certificate',
-      owner: 'Buckinghamshire',
       url: 'https://www.editor.planx.dev/buckinghamshire/apply-for-a-lawful-development-certificate/preview',
     },
-    session: {
-      source: 'PlanX',
-      id: '95f90e21-93f5-4761-90b3-815c673e041f',
-      createdAt: '2023-10-01 00:00:00',
-      submittedAt: '2023-10-02 00:00:00',
-    },
-    schema: {
-      url: 'https://theopensystemslab.github.io/digital-planning-data-schemas/v0.0.1/schema.json',
-    },
+    createdAt: '2023-10-01 00:00:00',
+    submittedAt: '2023-10-02 00:00:00',
```

Comment:
I think this is a question we can ultimately answer now: are `createdAt` / `submittedAt` redundant? Does an end-consumer actually care about the length of time between starting and submitting the application? (We don't send this to any back office systems currently)

Should this is simply be one required date that indicates when this payload is being submitted?

Reply:
This is a very fair comment. I'd vote for simplifying this and just capturing a "submittedAt" here 👍

Reply:
Updated accordingly!